xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -863,7 +889,7 @@ public void matchingVirtualHostDoesNotContainRouteAction() {
     VirtualHost virtualHost =
         VirtualHost.newBuilder()
             .setName("virtualhost00.googleapis.com")  // don't care
-            .addDomains("foo.googleapis.com")
+            .addDomains(TARGET_AUTHORITY)
```

Comment:
I found that using "foo.googleapis.com" actually it will fail/nack with a different reason: "No routes found", I think that's not what the test intended to test, so I changed to TARGET_AUTHORITY, namely "foo.googleapis.com:8080" instead.